cdap-ui/app/features/admin/controllers/namespace/settings-ctrl.js

```diff
@@ -45,10 +45,19 @@ angular.module(PKG.name + '.feature.admin')
 
     };
 
+    /**
```

Comment:
If we don't do steps 1,2,3, then the consequence is simply that when the namespace is recreated in the future, the user will then be able to see the templates, drafts, and dashboard, right?
(These 3 things aren't visible in other namespaces, right)?